docs/diagrams/AddCommandActivityDiagram.puml

```diff
@@ -0,0 +1,13 @@
+@startuml
+start
+:User enters Add command;
+
+if () then ([else])
```

Comment:
Do u think it is better to first have valid input, applicant already exists in TrackAScholar and then have else for invalid input?

Reply:
This one is purely for aesthetic purposes as the valid condition extends too long if placed as the first if condition on an activity diagram